tests/macros/macro_lib_a.html

```diff
@@ -0,0 +1,5 @@
+{% macro inner_impl() %}hello{% endmacro inner_impl -%}
+{# https://github.com/Keats/tera/issues/532
+   Try changing self:: to lib_a:: to pass one test and fail another.
```

Comment:
That way we'd see the lib_a import in the test number 3, but the test number 4 would start failing because the imported macro doesn't have lib_a visible there, only self.